crates/polars-arrow/src/compute/decimal.rs

```diff
@@ -9,12 +9,17 @@ fn leading_zeros(bytes: &[u8]) -> u8 {
 }
 
 fn split_decimal_bytes(bytes: &[u8]) -> (Option<&[u8]>, Option<&[u8]>) {
-    let mut a = bytes.split(|x| *x == b'.');
+    let mut a = bytes.splitn(2,|x| *x == b'.');
```

Comment:
What is the rationale of this change?

Reply:
@ritchie46 I suggested it to Chiel, the old code would erroneously accept `3.2.foobar` as a number since it split it in arbitrarily many pieces but only accessed the first two.

Reply:
Right. :+1: